File: Java/program71.c
///////////////////////////////////////////////////////
//
//  File name :     program71.c
//  Description :   Accept Input Number and Display Even Digits from that Number
//  Author :        Vaibhav
//  Date :          20/05/2025
//
///////////////////////////////////////////////////////

#include<stdio.h>

void DisplayEven(int iValue)
{

    if(iValue < 0)
    {
        iValue = -iValue;
    }
    while(iValue != 0)
    {        
        if( ((iValue % 10) % 2) == 0)
        {
            printf("%d\n",(iValue % 10));
        }
        iValue = iValue / 10;
    }
}

int main()
{
    int iNo = 0;

    printf("---- This Application is used to Display Even Digits from that Number ----\n");

    printf("Enter Number: ");
    scanf("%d",&iNo);

    printf("Even Digits in Number %d are : \n",iNo);
    DisplayEven(iNo);

    return 0;
}

File: program71.java
///////////////////////////////////////////////////////
//
//  File name :     program71.java
//  Description :   Accept Input Number and Display Even Digits from that Number
//  Author :        Vaibhav
//  Date :          
//
///////////////////////////////////////////////////////

import java.util.*;

class Digits
{
    public void DisplayEven(int iNo)
    {        
        if(iNo < 0)
        {
            iNo = -iNo;
        }
        while(iNo != 0)
        {
            if(((iNo % 10) % 2) == 0)
            {
                System.out.println(iNo % 10);
            }
            iNo = iNo / 10;                 
        }
    }
}

class program71
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        
        System.out.println("---- This Application is used to Display Even Digits from that Number ----");        

        System.out.println("Enter a number: ");
        int iValue = sobj.nextInt();


        Digits dobj = new Digits();
        
        dobj.DisplayEven(iValue);

    }
}